docs/reference/sdk-js.md

```diff
@@ -9,6 +9,10 @@ API from a JavaScript powered project.
 npm install @directus/sdk-js
 ```
 
+## TypeScript
```

Comment:
@juancarlosjr97 I would remove this snippet because I think is related with another PR, right?

Reply:
Yes, it is, I will remove it. Thank you @soyjimmysaenz